eamena/eamena/templates/views/forms/location.htm

```diff
@@ -350,9 +350,15 @@ <h5 id="arches-subjects-form" class="section-headline">{% trans "Overall Site Sh
                     <dd>
                         <div class="row">
                             <div class="col-xs-12">
+                                {% if resourcetypeid == 'HERITAGE_FEATURE.E24' %}
+                                <input style="width:33%; height: 36px; padding-top: 2px;" class="select2 arches-select2-crud-form resource-type hidden-xs" id="culturalp-dropdown" placeholder="{% trans 'Shape' %}" data-bind="select2: {value: getEditedNode('SITE_OVERALL_SHAPE_TYPE.E55', 'value'), dataKey: 'SITE_OVERALL_SHAPE_TYPE.E55'}"></input>
+                                
+                                <input style="width:33%; height: 36px; padding-top: 2px;" class="select2 arches-select2-crud-form resource-type  hidden-sm hidden-md hidden-lg" id="culperiod-dropdown" placeholder= "{% trans 'Shape' %}" data-bind="select2: {value: getEditedNode('SITE_OVERALL_SHAPE_TYPE.E55', 'value'), dataKey: 'SITE_OVERALL_SHAPE_TYPE.E55'}"></input>
+                                {% else %}
                                 <input style="width:33%; height: 36px; padding-top: 2px;" class="select2 arches-select2-crud-form resource-type hidden-xs" id="culturalp-dropdown" placeholder="{% trans 'Shape' %}" data-bind="select2: {value: getEditedNode('SITE_OVERALL_SHAPE_TYPE_NEW.E55', 'value'), dataKey: 'SITE_OVERALL_SHAPE_TYPE_NEW.E55'}"></input>
                                 
                                 <input style="width:33%; height: 36px; padding-top: 2px;" class="select2 arches-select2-crud-form resource-type  hidden-sm hidden-md hidden-lg" id="culperiod-dropdown" placeholder= "{% trans 'Shape' %}" data-bind="select2: {value: getEditedNode('SITE_OVERALL_SHAPE_TYPE_NEW.E55', 'value'), dataKey: 'SITE_OVERALL_SHAPE_TYPE_NEW.E55'}"></input>
+                                {% endif %}
```

Comment:
@mradamcox  the _NEW tag will disappear at the end of the data migration process (using the rename_entity_type utility: https://github.com/azerbini/eamena_dev/blob/15561846fab51b0a8dc89aebd3240d8e41482c26/arches/management/commands/package_utils/migrate_resources.py#L385) so this bit of code will become redundant then.